import-service/src/catalogBatchProcess.js

```diff
@@ -0,0 +1,78 @@
+const { Client } = require("pg");
```

Comment:
catalogBatchProcess might be created in Product service, as works with products and database.